src/smart-components/ResourcesCatalog.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
+const ResourcesCatalog = () => {
+  const { QuickStartCatalogPage } = useChrome();
```

Comment:
Apps can choose to add the catalog (or alternatively we can add this globally in insights-chrome?)